README.md

```diff
@@ -91,7 +91,7 @@ Wə kunən zə dus rēsīklērən vōr andərə klankən:
 * *c* vōr də kombinatīs *sj*, *sh* en *ch* (*cāl*, *campō* en *cōkōlādə*, mār *mesjə*);
 * *q* vōr də inhēmsə kombinatī *ng* (*op ən laqe baqk pasən vēl kindərən*), wat mə tot nog tö də vīsūēl mēst stōrəndə vərandəriq van al mïn vōrstelən lïkt;
 * *x* vōr də inhēmsə kombinatī *ch* (*glimlax*, *sxōl*), als ən knipōg nār het Grīksə alfābet;
-* *y* vōr soməgə üthēmsə *j* en *g* dī ongəvēr kliqkən als ‘zj’ (*yournāl*, *yīhād*, *barāyə*).
+* *y* vōr soməgə üthēmsə *j* en *g* dī ongəvēr kliqkən als ‘zj’ (*yurnāl*, *yīhād*, *barāyə*).
```

Comment:
Waarom zou je hier *yurnāl* schrijven in plaats van *yournāl*?
(In elk geval een aparte PR hiervoor.)

Reply:
Ik had nog niet gezien dat je later in je tekst spreekt over het fonetisch schrijven van alle uitheemse spellingswijzen. Het voelde al als een logisch gevolg van het fonetisch maken van de spelling in die paragraaf. Ik dacht oorspronkelijk dat het over het woord "journal" ging (en dan nog zou je over de 'r' kunnen discussiëren), maar er staat een lange a dus gaat het over "journaal" in welk geval ik denk dat het idd sowieso _yörnāl_  is, zoals de latere spelling.

Reply:
Als je die *yurnāl*-wijziging ongedaan maakt, aanvaard ik graag je PR.